addons/payment_authorize/models/payment_acquirer.py

```diff
@@ -0,0 +1,94 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import logging
+import pprint
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+from .authorize_request import AuthorizeAPI
+
+_logger = logging.getLogger(__name__)
+
+
+class PaymentAcquirer(models.Model):
+    _inherit = 'payment.acquirer'
+
+    provider = fields.Selection(
+        selection_add=[('authorize', 'Authorize.Net')], ondelete={'authorize': 'set default'})
+    authorize_login = fields.Char(
+        string="API Login ID", required_if_provider='authorize', groups='base.group_system')
+    authorize_transaction_key = fields.Char(
+        string="API Transaction Key", required_if_provider='authorize', groups='base.group_system')
+    authorize_signature_key = fields.Char(
+        string="API Signature Key", required_if_provider='authorize', groups='base.group_system')
```

Comment:
Probably worth extra comments and separation in the declaration of "public fields" vs "private fields", to remind all future eyes about what can and cannot be leaked. And I guess the `groups='base.group_system'` restriction on the public fields is unnecessarily confusing, especially as there is a controller that gives the same info publicly.

Acquirer records are supposed to be write-protected by the record-level ACLs anyway, so the field-level group restrictions shouldn't be used for that. They wouldn't protect against deletion for example.

Reply:
As discussed, I removed the `groups` attribute on public fields of all acquirers and added comments & tooltips where it was necessary.
The `groups` attribute is kept on fields holding private information as a safety net for too generous (custom) ACLs giving read rights on the `payment.acquirer` model.